app/services/storage/local_disk_service.rb

```diff
@@ -13,6 +13,12 @@ def get_file(filename)
       File.open(path, "r")
     end
 
+    def get_file_io(filename)
```

Comment:
Does this mean we've got `get_file` and `get_file_io` that are identical?

Reply:
Ah, I see, they're not quite identical in the s3 version. Cool.

Reply:
Ah, not anymore...